apps/api/pages/api/availability/_get.ts

```diff
@@ -134,7 +135,14 @@ async function handler(req: NextApiRequest) {
     where: { id: { in: allMemberIds } },
     select: availabilityUserSelect,
   });
-  if (!isAdmin) throw new HttpError({ statusCode: 403, message: "Forbidden" });
+  const memberRoles = team.members.reduce((acc, membership) => {
+    acc[membership.userId] = membership.role;
+    return acc;
+  }, {});
+  // check if the user is a team Admin or Owner, if it is a team request
```

Comment:
Nit: This comment shouldn't be added because you've named the variable well. It's redundant.